Assets/MRTK/SDK/Features/Utilities/Solvers/Follow.cs

```diff
@@ -374,22 +374,29 @@ public override void SolverUpdate()
             Quaternion goalRotation = Quaternion.identity;
             ComputeOrientation(goalPosition, wasClamped, ref goalRotation);
 
-            // Avoid drift by not updating the goal position when not clamped
-            if (wasClamped)
+            if (recenterNextUpdate)
             {
-                GoalPosition = goalPosition;
+                PreviousGoalRotation = goalRotation;
+                PreviousReferencePosition = goalPosition;
+                PreviousReferenceRotation = goalRotation;
```

Comment:
Do these not need to continue being set to `refPosition` and `refRotation`?

Reply:
Actually, look through the code...are `PreviousReferencePosition` and `PreviousReferenceRotation` actually used anywhere?

Reply:
Guess not!